exercises/all-your-base/src/lib.rs

```diff
@@ -43,4 +43,4 @@ pub fn convert(number: &[u32], from_base: u32, to_base: u32) -> Result<Vec<u32>,
         from_base,
         to_base
     )
-}
+}
```

Comment:
Please ensure files end with a trailing newline.

Reply:
Okay, thank you!